packages/odyssey-react/src/components/Text/Text.module.scss

```diff
@@ -0,0 +1,343 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.root {
+  color: $text-body;
+  font-family: $body-font-family;
+  font-size: ($base-font-size / 16px) * 100%;
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: 0;
+  line-height: $base-line-height;
+  font-feature-settings: "lnum", "pnum";
+  text-decoration-skip-ink: all;
+
+  /* stylelint-disable selector-max-type, selector-max-class, selector-class-pattern */
```

Comment:
I understand the need to escape `max-type` here but `max-class` and `selector-class-pattern` feel like rules we could keep in tact by refactoring our code here? Can you provide more context on why we need to remove these rules?

Reply:
- `selector-class-pattern` has to be removed because in [specific instances](https://github.com/okta/odyssey/pull/1076/files#diff-348f5695e010ebac8f4874c3ed3d5bc1ca1c1892257fd13be09361f1f8f4923fR81) we have selectors which are only one letter long, the error is thrown in these cases.
- `selector-max-class` is disabled because of we have a couple of deeply nested selectors, the error is thrown on the [instances you called out](https://github.com/okta/odyssey/pull/1076/files#r717506878)

Reply:
Thanks @arnoldsandoval-okta I see where you write out `styles[as]` to make use of these single line classes. Can we move the style lint escape to be more surgical where we keep track of these exceptions **only** where they are needed? That will help restore sanity around this file and make it easier to maintain